node/src/main/scala/com/wavesplatform/transaction/VersionedTransaction.scala

```diff
@@ -1,11 +1,19 @@
 package com.wavesplatform.transaction
 
 trait VersionedTransaction {
-  def version: TxVersion
+  val version: TxVersion
+  val maxVersion: TxVersion
```

Comment:
Moving `maxVersion` to `TransactionType` would better reflect the reality, since max version is a property of transaction type, rather than some particular transaction

Reply:
Refactored